json-serde/src/main/java/org/openx/data/jsonserde/JsonSerDe.java

```diff
@@ -302,22 +302,22 @@ public Object serializeField(Object obj,
                                             Boolean.FALSE;
                         break;
                     case BYTE:
-                        result = ((ByteObjectInspector)poi).get(obj);
+                        result = Byte.valueOf(((ByteObjectInspector)poi).get(obj));
                         break;
```

Comment:
Since result is an object, not a primitive, Java autoboxing takes care of that automatically.
See http://docs.oracle.com/javase/tutorial/java/data/autoboxing.html